vmdb/app/helpers/api_helper/results.rb

```diff
@@ -14,12 +14,24 @@ def add_href_to_result(hash, type, id)
       hash
     end
 
+    def add_parent_href_to_result(hash)
+      hash[:href] = "#{@req[:base]}#{@req[:prefix]}/#{@req[:collection]}/#{@req[:c_id]}"
```

Comment:
the segments are well know, no need for URI.  Also, if we want to use URI we'd want to change in other places too for consistency.